imagetest/test_suites/metadata/startup_script_test.go

```diff
@@ -103,21 +103,29 @@ func TestStartupScripts(t *testing.T) {
 			t.Fatalf("could not reinstall guest-agent: %s", err)
 		}
 	} else {
-		var cmd *exec.Cmd
+		var cmd,fallback *exec.Cmd
 		switch {
 		case utils.CheckLinuxCmdExists("apt"):
 			cmd = exec.Command("apt", "reinstall", "-y", "google-guest-agent")
 		case utils.CheckLinuxCmdExists("dnf"):
 			cmd = exec.Command("dnf", "-y", "reinstall", "google-guest-agent")
+			cmd = exec.Command("dnf", "-y", "upgrade", "google-guest-agent")
```

Comment:
minor typo: the second line probably should be fallback = exec.Command("dnf", "-y", "upgrade", "google-guest-agent")

Reply:
Woops, yeah.